### DISCUSSION TITLE
Installing GUI applications

### DISCUSSION THREAD

**Discussion by magnusviri** [pkgxdev/pkgx#429] [Ideas]
[VSCodium](https://github.com/VSCodium/vscodium) is a GUI opensource application. The developers do not offer a Mac download. The README says to use brew to install it. I am hoping that tea can also package and ship VSCodium.

**Comment by mxcl** [comment]
we're not doing GUI apps yet.

We only intend to install to `~/.tea` and as a CLI I don’t see how we can pkg `.app` stuff and have a good UX there.

Anyway, I went to the site and it looks like the provide darwin downloads to me.
